linux/jobs-and-processes-data-manipulation/file-streams/selecting-portions-of-a-line-using-cut.md

```diff
@@ -59,7 +59,7 @@ user2:/home/user2
 ... etc. etc. ...
 ```
 
-The fields in `/etc/passwd` are `:`-delimited with the first field containing the username and the second sixth field containing the user's home directory.
+The fields in `/etc/passwd` are `:`-delimited with the first field containing the username and the sixth field containing the user's home directory.
```

Comment:
hello @mickeypash and thanks for the contribution :tada: 

in this case, we should've remove the `sixth` word (no idea how that ended up there :sweat_smile: ) because the second field is the one that contains the user's home directory

```suggestion
The fields in `/etc/passwd` are `:`-delimited with the first field containing the username and the second field containing the user's home directory.
```